axelor-production/src/main/java/com/axelor/apps/production/service/configurator/ConfiguratorProdProcessServiceImpl.java

```diff
@@ -52,11 +56,29 @@ public ProdProcess generateProdProcessService(
     if (confProdProcess == null) {
       return null;
     }
+    String name;
     String code;
     StockLocation stockLocation;
     StockLocation producedProductStockLocation;
     StockLocation workshopStockLocation;
 
+    if (confProdProcess.getDefNameAsFormula()) {
+      Object computedName =
+          configuratorService.computeFormula(confProdProcess.getNameFormula(), attributes);
+      if (computedName == null) {
+        throw new AxelorException(
+            confProdProcess,
+            TraceBackRepository.CATEGORY_INCONSISTENCY,
+            I18n.get(
+                String.format(
+                    IExceptionMessage.CONFIGURATOR_PROD_PROCESS_INCONSISTENT_NAME_FORMULA,
+                    confProdProcess.getId())));
+      } else {
+        name = String.valueOf(computedName);
+      }
+    } else {
+      name = confProdProcess.getName();
```

Comment:
here, due to the change of this commit, `confProdProcess.getName()` can return `null`, but the field `name` is still required in prod process and it will throw an exception difficult to understand for the user. We have to add a check to throw an exception if `name` is null.